cypress/e2e/singleMovie_spec.cy.js

```diff
@@ -0,0 +1,22 @@
+describe('Single Movie Page', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3000/')
+       .get('.mini-poster').first().click()
+    })
+
+    it('A user should see the movie details for a specific movie', () => {
+      cy.url().should('include', '528085')
+        .get('h2').contains('2067')
+        .get('.single-movie-backdrop').should('have.attr', 'src', "https://image.tmdb.org/t/p/original//5UkzNSOK561c2QRy2Zr4AkADzLT.jpg")
+        .get('p').contains('Overview: A lowly utility worker is called to the future by a mysterious radio signal, he must leave his dying wife to embark on a journey that will force him to face his deepest fears in an attempt to change the fabric of reality and save humankind from its greatest environmental crisis yet.')
```

Comment:
could just update all of these to cy.contains('p', 'text') as well and remove the 'get' portion. It is redundant I think as written